flambe/cluster/const.py

```diff
@@ -3,7 +3,7 @@
 
 # SSH connection settings
 RETRY_DELAY = 1
-RETRIES = 60
+RETRIES = 300
```

Comment:
It takes longer for GCP GPU instances using their standard pytorch image to become available as the image for some reason doesn't come with the Nvidia drivers and instances need to install them when they come up. So I had to increase the retry number. It typically takes about 2-3 minutes. The workaround is either to create an image with the nvidia drivers pre-installed, or use another publicly available image that comes with the driver.